### PR TITLE
Bug 1215463 - [TV] Disallow new sync users. r=yifan

### PR DESCRIPTION
**Pull request by ferjm**

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=c4d74afa77a072841eae708996f8d0ef856d0e49

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=33a3163d54c0ef0482b9c98f1919d9fe7ecf175f

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=7f2e26225e4a736b35d689be083c4c6650780ea4

File: tv_apps/browser/js/sync/settings.js [hunk 1]
```diff
@@ -109,6 +113,34 @@
...
+     *   2. Logged in screen. We show to the user several different options:
+     *     + The user owning the Sync account. If the account belongs to a
+     *     verified user that previosly used Sync, we are fine, if not, we
+     *     show a warning asking the user to go to other Firefox client to
+     *     create a FxA and login into Sync.
+     *     + A button to disable sync. Clicking it will log the user out from
+     *     FxA and will clear the synchronized data.
+     *     + A button to start a sync on demand. Clicking it will disable the
+     *     button and the collections switches while the synchronization is
+     *     being performaned. Once the sync is done successfully, we enable
```
Comment (begeeben) [inline tv_apps/browser/js/sync/settings.js]: Should it be ```performed```?

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=833796c0b3198f69817c2bdbb5a2616f50f1745a

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=7f4b8698f96819a8881b4a42b27811c2ef9e4103

**Pull request merged**